modules/openapi-generator/src/main/java/org/openapitools/codegen/utils/PostProcessUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
@lucafuji new files added to openapi-generator project doesn't require this line: https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license

Reply:
removed this line